automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -142,6 +142,16 @@ trait Orchestration extends FireCloudClient with LazyLogging {
     postRequestWithMultipart(Config.FireCloud.orchApiUrl + "api/workspaces/" + ns + "/" + wsName + "/" + "importEntities", fileName, fileContent)
   }
 
+  def importMethodConfigFromRepo(ns: String, wsName: String, configurationNamespace: String, configurationName: String, configurationSnapshotId: Int, destinationNamespace: String, destinationName: String)(implicit token: AuthToken) = {
```

Comment:
This should go in a new `methodConfigs` object to provide a structure similar to what we expose through swagger. I know we still have a couple of outliers that need to be relocated... sorry about that.

Also, I think it's helpful for these API methods to log what they're doing to help with debugging. Again, I'm sorry that we haven't yet done this universally. I've added both of these clean-up tasks to my TODO list.